### PR TITLE
Align code snippets

### LINKED ISSUE #1724: Optimize command examples (Service)

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?
This PR resolves #1724

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 1]
```diff
@@ -164,21 +162,31 @@ You can choose to create a custom database with a different user and password. T
...
 brew tap homebrew/services
+----
+
+----
 brew install mysql@8.0
+----
+
+----
 brew services start mysql@8.0
 ----
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:192]: Snippets need an update on lines 200, 208, 212, 218, and 222.

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 2]
```diff
@@ -134,20 +131,26 @@ sudo ufw allow 9901
...
+
+
+----
 cd lisk-service
+----
+
+
+----
 git checkout v0.7.0
 ----
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:147]: I would format it like this:
```suggestion
. Clone Lisk Service repository.
+
----
git clone https://github.com/LiskHQ/lisk-service.git
----
. Change working directory to the new repository.
+
----
cd lisk-service
----
. Switch to the release branch.
+
----
git checkout v0.7.0
----
```

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 3]
```diff
@@ -303,7 +305,7 @@ It is strongly recommended that you synchronize your Lisk Core node with the net
 
 If you have not already done so, clone the {url_github_service}[lisk-service^] GitHub repository and then navigate into the project folder and check out the latest release.
 
-[source,bash]
+
 ----
 # Clone Lisk Service repository
 git clone https://github.com/LiskHQ/lisk-service.git
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:311]: The source needs an update. Same suggestion for this as mentioned above for the `setup/docker.adoc`.

File: docs/antora/modules/ROOT/pages/management/docker.adoc [hunk 4]
```diff
@@ -157,9 +156,11 @@ Log out of the docker container by pressing kbd:[CRTL] + kbd:[D].
...
 Reset the databases for Redis after dropping the MySQL database:
 
-[source,bash]
 ----
 sudo docker exec -it lisk-service_redis_core_persistent_1 /bin/sh
+----
+
+----
 redis-cli flushall
 ----
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/management/docker.adoc:165]:
```suggestion
.Execute command in running docker container.
----
sudo docker exec -it lisk-service_redis_core_persistent_1 /bin/sh
----

.Truncate redis database.
----
redis-cli flushall
----
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looking good!

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 5]
```diff
@@ -134,20 +131,26 @@ sudo ufw allow 9901
...
 
 == Build Lisk Service
 
 To build Lisk Service from local files, first, navigate to the `lisk-service` repository on your system:
 
-.Working directory: ./lisk-service/
-[source,bash]
+.Working directory: lisk-service/
```
[the remark this answers is not in the document]
Comment (Tschakki) [reply docs/antora/modules/ROOT/pages/setup/docker.adoc:153]: `./` indicates the current folder, so imho it doesn't fit here in the caption. It assumes the user is one folder up from the `lisk-service` folder, but in fact the user should be inside the `lisk-service` folder, that's why i would remove it

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 6]
```diff
@@ -134,20 +131,26 @@ sudo ufw allow 9901
...
 git clone https://github.com/LiskHQ/lisk-service.git
+----
+
+
+----
 cd lisk-service
+----
+
+
+----
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:145]:
```suggestion

----
git clone https://github.com/LiskHQ/lisk-service.git
----

----
cd lisk-service
----

Switch to the latest release branch.

----
```

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 7]
```diff
@@ -112,13 +111,11 @@ NVM is a bash script that enables the management of multiple active Node.js vers
 . Install v16 of Node.js using NVM.
```
[the remark this answers is not in the document]
Comment (Tschakki) [reply docs/antora/modules/ROOT/pages/setup/source.adoc:111]: Good catch, I created a corresponding issue: #1768

File: docs/antora/modules/ROOT/pages/setup/docker.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (TalhaMaliktz) [reply docs/antora/modules/ROOT/pages/setup/docker.adoc:153]: I see, In that case, I also would update lines 173, 201, and 209 to align content.

**Review by TalhaMaliktz — APPROVED** [review]
Thank you! Just a minor change! rest LGTM

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Pull request merged**